.azure-pipelines/windows.sh

```diff
@@ -75,73 +71,82 @@ else
 fi
 
 ################################################################################
-# Build DMD
+# Build DMD (incl. building and running the unittests)
 ################################################################################
 
-DMD_BIN_PATH="$DMD_DIR/generated/windows/release/${MODEL}/dmd"
+DMD_BIN_PATH="$DMD_DIR/generated/windows/release/$MODEL/dmd"
 
-cd "${DMD_DIR}/src"
-"${DM_MAKE}" -f "${MAKE_FILE}" reldmd-asserts DMD="$DMD_BIN_PATH" MAKE="$DM_MAKE"
-
-################################################################################
-# WORKAROUND: Build zlib separately with DigitalMars make
-# REASON: whitespace path variables in DigitalMars make from indirect invocation from Phobos
-################################################################################
-
-if [ "$MODEL" != "32" ] ; then
-    cd "${DMD_DIR}/../phobos/etc/c/zlib"
-    "${DM_MAKE}" -f win64.mak MODEL=${MODEL} "zlib${MODEL}.lib" "CC=$CC" "LIB=$AR" VCDIR=. MAKE="$DM_MAKE"
-fi
+cd "$DMD_DIR/src"
+"$DM_MAKE" -f "$MAKE_FILE" MAKE="$DM_MAKE" BUILD=debug unittest
+DFLAGS="-L-LARGEADDRESSAWARE" "$DM_MAKE" -f "$MAKE_FILE" MAKE="$DM_MAKE" reldmd-asserts
```

Comment:
`/LARGEADDRESSAWARE` was needed to overcome process memory limits when compiling the Phobos unittests with a 32-bit DMD (before hitting the max 32K symbols in OMF object file limitation). The leading dash somehow led to something prepending a path (=> something like `/C/Program Files/Git/LARGEADDRESSAWARE`), hence the `-`, which fortunately works for both optlink.exe and link.exe.